SConstruct

```diff
@@ -85,15 +84,12 @@ def update_rpm_version(version, tag):
             spec[line_num] = "Release:       {}%{{?relval}}%{{?dist}}\n".\
                              format(release)
         if line == "%changelog\n":
-            rpmdevpackager = find_executable('rpmdev-packager')
-            if not rpmdevpackager:
-                print("No rpmdev-packager found in path.")
-                return False
+            cmd = 'rpmdev-packager'
             try:
-                packager = subprocess.Popen(
-                    rpmdevpackager, stdout=subprocess.PIPE).communicate(
-                    )[0].strip().decode('UTF-8')
+                pkg_st = subprocess.Popen(cmd, stdout=subprocess.PIPE) # nosec
+                packager = pkg_st.communicate()[0].strip().decode('UTF-8')
             except OSError:
+                packager = pkg_st.communicate()[0].strip().decode('UTF-8')
```

Comment:
I can't actually see where this is used in this exception case. We're returning below, before the variable can be used. Am I missing something?

Reply:
@kjacque, Thanks for the comment: 
packager is used on the L95, L97 print command. 
 91             except OSError:
 92                 packager = pkg_st.communicate()[0].strip().decode('UTF-8')
 93                 print("You need to have the rpmdev-packager tool (from the "
 94                       "rpmdevtools RPM on EL7) in order to make releases.\n\n"
 95                       "Additionally, you should define %packager in "
 96                       "~/.rpmmacros as such:\n"
 97                       "%packager        John A. Doe <john.doe@intel.com>"
 98                       "so that package changelog entries are well defined")